scripts/src/java/org/oppia/android/scripts/ci/ComputeAffectedTests.kt

```diff
@@ -103,10 +103,18 @@ private fun computeAffectedTargetsForNonDevelopBranch(
   println("Affected test targets due to transitive build deps: $transitiveTestTargets")
 
   val allAffectedTestTargets = (affectedTestTargets + transitiveTestTargets).toSet()
+
+  // Filtering out the targets to be ignored.
```

Comment:
```suggestion
  /** Filtering out the targets to be ignored. */
```

Reply:
@anandwana001 I don't think that suggestion is correct. We should be using single-line (``//``) or block comments (``/* */``) in this context, not KDocs (``/** */``). KDocs are only used for documenting members, not within blocks of code.

Reply:
As per my understanding, the kdoc lint chek should fail as it required kdoc for all non-private fields and the doc should be under `/** */` not `//`. 

Looks like test is passing so I misunderstand that thing.